appveyor/Push-PrefastResultsToAppveyor.ps1

```diff
@@ -0,0 +1,8 @@
+$Dir = get-childitem .\ -Recurse -Filter "*.prefast.sarif"
+
+for ($i=0; $i -lt $Dir.Count; $i++)
+{
+  $fileLocation = $Dir[$i].FullName
+  Push-AppVeyorArtifact $fileLocation
+  .\appveyor\Convert-FileUrisToGitHubUris.ps1 $fileLocation
```

Comment:
So right now, I'm pushing the results before conversion and after conversion. There is another push artifact script at the end of the Convert-Files script